projects/FOCIL_Nimbus_CL.md

```diff
@@ -0,0 +1,127 @@
+# Implementation of FOCIL in Nimbus Cl
+
+Bringing Fork-Choice Enforced Inclusion Lists to the Nimbus Consensus Layer .
+(Execution work left as an optional stretch goal)
+
+### Collaborators 
+
+- Tamaghna (@razorclient)
+- Akash (@h3lio5)
+
+We will be persuing this work together and divide work as we see fit with blockers and work items
+
+
+
+## Motivation
+
+Fork-Choice Enforced Inclusion Lists (FOCIL) represent a protocol-level mechanism to guarantee timely and censorship-resistant transaction inclusion by embedding inclusion constraints directly into Ethereum’s fork-choice rule. 
+
+Basically:
+- **FOCIL (EIP-7805)** restores censorship-resistance by letting a 16-validator committee publish inclusion lists (ILs) each slot
+- blocks that omit valid IL transactions receive no fork-choice weight.
+
+The Nimbus Cl does not have an implemntation of this proposed changed implemented in its client so we propose to persue the implement for the the aferomentioned during this during EPF cohort
```

Comment:
```suggestion
The Nimbus CL does not have an implementation of this proposed change so we propose to pursue to implement the aforementioned during this during EPF cohort.
```

Reply:
it's not a big issue to be grammatically correct but many sentences have so many typos its hard to read)